src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -287,40 +287,30 @@ export class NetCoreDebugHelper implements DebugHelper {
         return pathNormalize(result, platformOS);
     }
 
-    private async installDebuggerInContainer(containerName: string, installPath: string): Promise<string> {
+    private async copyDebuggerToContainer(containerName: string, containerDebuggerDirectory: string): Promise<string> {

```

Comment:
Why does this method return a string, but we don't use it?

Reply:
Now it is being used.

---
In reply to: [400957167](https://github.com/microsoft/vscode-docker/pull/1800#discussion_r400957167) [](ancestors = 400957167)